### DISCUSSION TITLE
build for Alpine Linux/musl

### DISCUSSION THREAD

**Discussion by will-path** [biomejs/biome#747] [Ideas]
Hi,

I would like to use biome in my alpine container for local development, but it seems you all are not building that binary. Is it possible?

**Comment by ematipico** [comment]
Could you help? We would like to add support for Alpine, but we don't have enough knowledge and bandwidth to do so

**Comment by nhedger** [comment]
https://github.com/biomejs/biome/issues/823

**Comment by Conaclos** [comment]
It is now supported.
